commons-math-legacy/src/test/java/org/apache/commons/math4/legacy/analysis/interpolation/BicubicInterpolatingFunctionTest.java

```diff
@@ -286,6 +287,231 @@ public double value(double x, double y) {
                           maxTolerance,
                           false);
     }
+    
+    /**
+     * Test for partial derivatives of {@link BicubicSplineFunction}.
```

Comment:
`BicubicSplineFunction` does not exist; it should be `BicubicFunction`

Reply:
Done!